### DISCUSSION TITLE
Request For Yosys Installation steps for Centos 7

### DISCUSSION THREAD

**Discussion by Kaanth** [YosysHQ/yosys#4099] [Q&A]
Hi All,
     Greeting for the  New Year!!!
   
   Please provide the Installation steps/information for the Yosys tool in Cent-OS 7. It would be great help for me.

  Thanks in Advance.
  Kanth

**Comment by mmicko** [comment]
Hi,

If you also need other open source FPGA related software best we can suggest is to use OSS-CAD-Suite and download latest release for linux-x64 at https://github.com/YosysHQ/oss-cad-suite-build/releases/latest 
If you wish to compile yourself, dependencies are mentioned in README but package names differs per Linux distribution and package manager used so we shared instructions for Ubuntu/Debian as most commonly used.

Hope this helps, and all the best to you as well.

**Reply by Kaanth** [reply]
Thanks mmicko!!!
